specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/examples/CreateAScaleSetWithExtensionsTimeBudget.json

```diff
@@ -0,0 +1,183 @@
+﻿{
```

Comment:
You need add the link of this example file to compute.json file similar to below:

"$ref": "./examples/CreateAnAvailabilitySet.json"